dataloader/dataloader_constants.py

```diff
@@ -87,4 +87,4 @@
 GRAPH_COLLECTIONS_CATEGORIES = "collections_categories"
 
 
-COLLECTION_REGEX = "^(pli-tv-b[ui]-vb|XX|OT|[A-Z]+[0-9]+|[a-z\-]+)"
+COLLECTION_REGEX = "^(pli-tv-b[ui]-vb|XX|OT|NG|NK[^.]+|(?![N])[A-Z]+[0-9]+|[a-z\-]+|)"
```

Comment:
Can we keep the regex consistent with each other i.e. the same as in main.py? (also, the | at the end is not needed at all)